client/src/commands/UserInputUtil.ts

```diff
@@ -632,6 +632,37 @@ export class UserInputUtil {
 
     }
 
+    public static async saveDirectory(placeHolder: string): Promise<vscode.Uri> {
```

Comment:
This function is very similar to the browseEdit one, can we create more of a common function?

Reply:
I've changed the `browseEdit` function and allowed it to be more flexible.